cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -575,7 +575,10 @@ func kubeClientGet(uc *UpstreamController, namespace string, name string, queryT
 		return nil, err
 	}
 	if err == nil && obj != nil {
-		util.SetMetaType(obj.(runtime.Object))
+		err := util.SetMetaType(obj.(runtime.Object))
+		if err != nil {
+			klog.Warningf("set MetaType error %v for obj %v, %v", err, queryType, obj.GetName())
+		}
 	}
 	return obj, err
```

Comment:
Can the upper layer process guarantee that when `err == nil`, obj must be non-nil.
By the way, there is no need to output the log here, because the log is already output after the method is called.
ref: https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/edgecontroller/controller/upstream.go#L598

```golang
object, err := kubeClientGet(uc, namespace, name, queryType, msg)
if errors.IsNotFound(err) {
	klog.Warningf("message: %s process failure, resource not found, namespace: %s, name: %s", msg.GetID(), namespace, name)
	return
}
if err != nil {
	klog.Warningf("message: %s process failure with error: %s, namespace: %s, name: %s", msg.GetID(), err, namespace, name)
	return
}
```

Reply:
But we can't log error of function util.SetMetaType()